src/dom-updates.js

```diff
@@ -221,7 +233,7 @@ const showConfirmedBooking = (room, date) => {
     body: JSON.stringify(bookRoomReceipt),
     headers: { 'Content-Type': 'application/json' }
   })
-    .then(response => response.json())
+    .then(response => errorHandle(response))
```

Comment:
Why'd you put the POST in `dom-updates`?

Reply:
Because I can't get it to work in the other file... Something I'm still trying to troubleshoot.

Reply:
You'll see the code commented out where the post function is in the API file. Still trying to troubleshoot this, so ignore it for now.

Reply:
K gonna take an extra look at this here in a bit